### ISSUE TITLE
Include patch number in semantic versioning

### ISSUE THREAD

**Issue by gabro** [ReactiveCocoa/ReactiveCocoa#2201]
Hi and first of all thank you very much for the awesome library!

I've recently become one of the maintainers of the ReactiveCocoa spec file for CocoaPods, and I've already pushed beta 9 and RC1.

Unfortunately I noticed an issue with the ordering of versions.
CocoaPods uses strict [semantic versioning](http://semver.org/) for determining the ordering of specs and the latest 3.0 releases lack the patch version number.

Right now, version `3.0.0-alpha.1` is considered the current one because for that version @ashfurrow took the effort of using its own fork for "fixing" the version in the tag to include the patch number.

This behavior [has been confirmed](https://github.com/CocoaPods/cocoapods.org/issues/185) by one of the maintainers of CocoaPods.

While I understand you guys don't officially support CocoaPods, many people use it and this minor issue with the version ordering can lead to [people using an outdated version](https://github.com/ReactiveCocoa/ReactiveCocoa/issues/2166) (in this case the first alpha of RAC 3) and probably opening many issues about it.

Long story short, can you please add the patch number to the future versions of RAC in order to fully comply with semver?

**Comment by ashfurrow** [comment]
The CocoaPods version and the tag can be out-of-sync. Maybe the right approach would be inferring the correct semver for the pod from the tag. Not sure how RC's work on semver – @orta?

**Comment by orta** [comment]
The RC/alpha/beta stuff will be fine, I think it would probably make more sense to do what @ashfurrow says by de-syncing the CP version from the tag. You'd have to push a new version ( or deprecate the `3.0-RC1` ) so might be worth waiting for an RC2 if it's going to happen.

**Comment by ashfurrow** [comment]
Sounds fine to me. I mean, I don't know if RC's, alphas, or betas should be "current" – like, `pod 'ReactiveCocoa'` shouldn't download prerelease software, right? This could just be something that we live with until RAC 3 is out for realsies.

**Comment by gabro** [comment]
@ashfurrow, @orta I tried pushing a `3.0.0-RC.1` pointing to the `v3.0-RC.1` tag but it didn't pass the validation

> Validating podspec
>  -> ReactiveCocoa (3.0.0-RC.1)
>    - WARN  | The version should be included in the Git tag.

Is there something I'm missing

**Comment by ashfurrow** [comment]
Ha, yeah I've gotten that error before. Trunk takes any warning as a failure. Using `--allow-warnings` should fix that.

**Comment by gabro** [comment]
@ashfurrow oh I didn't know you could cheat :P

Pushing a new version right now!

**Comment by gabro** [comment]
@ashfurrow mmh, apparently `--allow-warnings` doesn't allow warnings...

<img width="485" alt="screen shot 2015-07-21 at 5 25 28 pm" src="https://cloud.githubusercontent.com/assets/691940/8804658/8c354222-2fcd-11e5-9ea2-b88e99e7d5f4.png">

**Comment by ashfurrow** [comment]
Hmm weird. Wanna email me the podspec? ash@ashfurrow.com

**Comment by gabro** [comment]
You got it

**Comment by ashfurrow** [comment]
I'll follow up with the CocoaPods team and let you know :tada:

**Comment by gabro** [comment]
Awesome, thanks!

**Comment by ashfurrow** [comment]
This appears to be a problem on the server-side podspec validation. 

![screen shot 2015-07-21 at 12 00 18 pm](https://cloud.githubusercontent.com/assets/498212/8805599/1a56adda-2fa0-11e5-980d-9dce132aedc2.png)

I'm chatting with the CocoaPods people now.

**Comment by gabro** [comment]
I see, I've also opened an issue on CocoaPods. Thanks for the help https://github.com/CocoaPods/CocoaPods/issues/3855#issuecomment-123380517
